File: domain/src/main/java/com/aceplus/domain/model/forApi/OutletStockAvailability.java
package com.aceplus.domain.model.forApi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by phonelin on 3/21/17.
 */

public class OutletStockAvailability {

    @SerializedName("OutletStockAvailabilityNo")
    @Expose
    private String outletStockAvailabilityNo;
    @SerializedName("Date")
    @Expose
    private String date;
    @SerializedName("CustomerId")
    @Expose
    private Integer customerId;
    @SerializedName("OutletStockAvailabilityItem")
    @Expose
    private List<OutletStockAvailabilityItem> outletStockAvailabilityItem = null;

    public String getOutletStockAvailabilityNo() {
        return outletStockAvailabilityNo;
    }

    public void setOutletStockAvailabilityNo(String outletStockAvailabilityNo) {
        this.outletStockAvailabilityNo = outletStockAvailabilityNo;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Integer getCustomerId() {
        return customerId;
    }

    public void setCustomerId(Integer customerId) {
        this.customerId = customerId;
    }

    public List<OutletStockAvailabilityItem> getOutletStockAvailabilityItem() {
        return outletStockAvailabilityItem;
    }

    public void setOutletStockAvailabilityItem(List<OutletStockAvailabilityItem> outletStockAvailabilityItem) {
        this.outletStockAvailabilityItem = outletStockAvailabilityItem;
    }



}

File: domain/src/main/java/com/aceplus/domain/entity/outlet/OutletStockAvailability.kt
package com.aceplus.domain.entity.outlet

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "outlet_stock_availability")
class OutletStockAvailability {

    @PrimaryKey
    @ColumnInfo(name = "outlet_stock_availability_id")
    @SerializedName("outlet_stock_availability_id")
    @Expose
    var outlet_stock_availability_id: Int = 0

    @ColumnInfo(name = "customer_id")
    @SerializedName("customer_id")
    @Expose
    var customer_id: Int = 0

    @ColumnInfo(name = "date")
    @SerializedName("date")
    @Expose
    var date: String? = null

}